### PR TITLE
Higher Test Coverage.

### LINKED ISSUE #3: Test coverage is too low
There are less than 1 test per function. I don't like it

### PR DESCRIPTION
**Pull request by devyn-miller**
I ran coverage report (you'll needa    pip install coverage)

### CODE REVIEW

**Comment by devyn-miller** [comment]
image

**Review by ProvenNoIQ — APPROVED** [review]
lgtm

**Review by av-chapman — APPROVED** [review]
Good job!

**Review by jordanbruce44 — APPROVED** [review]
LGTM

**Review by tristntran — CHANGES_REQUESTED** [review]
Please someone add docstrings to these tests

**Comment by annagrigoryan7** [comment]
lgtm i will work on adding docstrings to these tests

**Review by annagrigoryan7 — APPROVED** [review]

**Review by katherinehansen1 — CHANGES_REQUESTED** [review]
There are two functions named test_create_state_map() (lines 57 and 71). Do we need both of these? Can we merge them into one?

**Review by katherinehansen1 — CHANGES_REQUESTED** [review]
I also think we should be consistent with having one test function that contains multiple assertions per function, rather than have several functions with one single test per function. Right now its a little hard to follow, but I think having one test function per function in gamblers_ruin.py would be easier!

**Review by jordanbruce44 — APPROVED** [review]
LGTM. I'd say we push these through so we have less commits to combine. Makes it a bit more thorough and clean, but not necessarily done.

**Review by ProvenNoIQ — APPROVED** [review]
lgtm

**Comment by tristntran** [comment]
Comments are now good. They have some whitespace issues causing tests to fail

**Review by tristntran — APPROVED** [review]
Like the docstrings. looks good at a first pass. I'm not doing any checks on validity of tests. I'll do that when I grade it :), so you should all do a double check yourselves before merging.

**Review by katherinehansen1 — APPROVED** [review]
